src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkAddressChangedTests.cs

```diff
@@ -13,13 +13,15 @@ public partial class NetworkChangedTests
         private readonly NetworkAddressChangedEventHandler _addressHandler = delegate { };
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.FreeBSD)]
```

Comment:
Should we start making these ActiveIssues instead?

Reply:
yes, I can write one up. Or we can log it against #24386 as unfinished work.